scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,9 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    brent does not ensure that the located minimum is within the initially
```

Comment:
Yeah, exactly. Just saw that.

But the input could be a pair or a triple. If it is a pair, the statement is true, right? (The documentation says that too)